tests/evm_backends/pyevm_env.py

```diff
@@ -156,15 +154,15 @@ def _check_computation(self, computation):
     def get_code(self, address: str):
         return self._state.get_code(_addr(address))
 
-    def time_travel(self, num_blocks=1, time_delta: int | None = None) -> None:
+    def time_travel(self, num_blocks=1) -> None:
         """
         Move the block number forward by `num_blocks` and the timestamp forward by `time_delta`.
         """
 
         # Cast since ExecutionContextAPI does not have the properties we need to change
         context = cast(ExecutionContext, self._state.execution_context)
```

Comment:
i'm not a huge fan of `cast()`. we should either `assert isinstance (self._state.execution_context, ExecutionContext)` or just squash the mypy errors with `pragma: notype`

Reply:
It's just for documentation actually, so I don't wanna have the notype. Assert is fine!